tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts

```diff
@@ -70,22 +70,32 @@ export default {
         TwitterValidationData: {
             _enum: {
                 PublicTweet: "PublicTweet",
-                OAuth2: "OAuth2",
+                OAuth2: "TwitterOAuth2",
             },
         },
         PublicTweet: {
             tweet_id: "Vec<u8>",
         },
-        OAuth2: {
+        TwitterOAuth2: {
             code: "Vec<u8>",
             state: "Vec<u8>",
             redirect_uri: "Vec<u8>",
         },
         DiscordValidationData: {
+            _enum: {
+                PublicMessage: "PublicMessage",
+                OAuth2: "DiscordOAuth2",
+            },
+        },
+        PublicMessage: {
```

Comment:
`DiscordPublicMessage` or `DiscordChannelMessage` (I find latter more accurate) ? Just to avoid future rename like it was with `OAuth2` -> `TwitterOAuth2`